src/nft/components/collection/CollectionNfts.tsx

```diff
@@ -530,7 +526,7 @@ export const CollectionNfts = ({ contractAddress, collectionStats, rarityVerifie
         </Box>
       </AnimatedBox>
       <InfiniteScroll
-        next={() => (isNftGraphQl ? loadNext(DEFAULT_ASSET_QUERY_AMOUNT) : fetchNextPage())}
+        next={() => (isNftGraphQl ? loadMore() : fetchNextPage())}
```

Comment:
```suggestion
        next={isNftGraphQl ? loadMore : fetchNextPage}
```

assuming `loadMore` and `fetchNextPage` are memoized / referentially stable across renders, this would also keep the value passed to `next` referentially stable